integration/prep.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+ME="$(readlink -f "$0")"
+HERE="$(dirname "$ME")"
```

Comment:
Is this similar to the current working-directory?

Reply:
Not really, instead it refers to the directory containing the script being executed.

So, if you're executing the script from _within_ the directory the script sits in, then yes: this value and $(pwd) would return similar paths (though there might still be relative vs absolute issues).

But in the case where you execute this script from a location _outside_ the script directory (say, by running something like `./../this/is/not/in/pwd/script.sh`) then $(pwd) and "$HERE" will be very different.